tests/plugins/copyformatting/manual/nestededitablefocus.html

```diff
@@ -0,0 +1,64 @@
+<style>
+#square {
+	margin-top: 20px;
+	width: 100px;
+	height: 100px;
+	border: 1px #000 solid;
+	background-color: red;
```

Comment:
This style is duplicated on `blur`, so maybe inline it.
```suggestion
```

Not sure tho if suggestion completely removes line.

Reply:
I don't see much sense in moving it directly into element. The outcome should be the same, yet putting default styles into stylesheet seems more readable to me.